site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -313,7 +313,7 @@
     * @param {String} button_id the ID of the submit button to be enabled / disabled
     * @param (String) error_div_id the ID of the error message to be displayed if submit is disabled
     */
-    function updateSubmitButton(valid_grader, valid_students, new_grader, grader_removed, button_id, error_div_id) {
+    function updateSubmitButton(valid_grader, valid_students, new_grader, grader_removed, button_id, error_div_id, self_grading) {
```

Comment:
Shouldn't self_grading also be added to the comment above documenting each of the function arguments?